controllers/storagecluster/external_resources.go

```diff
@@ -739,6 +763,18 @@ func (r *StorageClusterReconciler) enableCsiDrivers(availableSCCs []StorageClass
 
 	}
 
+	// Read topology-failure-domain-label from external resources and update ConfigMap
+	if instance.Spec.ExternalStorage.Enable {
+		topologyDomainLabel, err := r.getTopologyFailureDomainConfig(instance.UID)
+		if err != nil {
+			r.Log.Error(err, "failed to get topology failure domain config from external resources")
+			return err
+		}
+		if topologyDomainLabel != "" {
+			clientConfig.Data["TOPOLOGY_FAILURE_DOMAIN_LABEL"] = topologyDomainLabel
```

Comment:
all the keys in the client-op config uses camel case can we have this key in camel case and as a constant as well?

Reply:
@subhamkrai i think we need to change it as its the key in the ocs-client-operator configmap https://github.com/red-hat-storage/ocs-client-operator/pull/495/files#r2517276668

it should be `topologyFailureDomainLabel`

Reply:
if this is multiple labels, we need to adjust the key in the configmap  `topologyFailureDomainLabels`